### ISSUE TITLE
Please remove approval on Crowdin

### ISSUE THREAD

**Issue by maboroshin** [fast4x/RiMusic#3483]
### Steps to reproduce the bug

1. Try the translation on Crowdin.
2. Existing translations have been approved.
3. So, it is impossible to overwrite translations

### Expected behavior

Please remove approval for Japanese translation. Otherwise, it can't be modified or improved.

### Actual behavior

For example, "About" is approved and locked as it is in English. 

![image](https://github.com/user-attachments/assets/742b19a9-2075-4a5f-845f-448507377c48)

Japanese "アプリの情報" is not applicable as it has not been approved.

### Additional information

Approval is an assumption that is 100% correct. Some translators are novice translators or use machine translation. In that case, it is impossible to be 100% correct. And mistakes and misunderstandings can happen even if we are used to translating. Approval locks the strings.

In other words, to approve a language that one does not understand is to include a lock on the wrong translation. Another translator comes along who notices the mistranslation. New translators report problems with translations not being reflected. Because the approval locks in the mistranslation.

Even without an approval system by reviewers, Crowdin has a voting system. Because reviewers may be gone. If there is no approval system, simply a new translator can continue to make improvements. If there are any irregularities in this vote, the translation will need to be finalized with approval.

Related issues:
* Ukrainian? https://github.com/fast4x/RiMusic/issues/2456
* Brasilian portuguese https://github.com/fast4x/RiMusic/issues/2127#issuecomment-2143465416

**Comment by fast4x** [comment]
Hi, i've added to you Proofreader permission in Japanese lang, now can improve or approve

**Comment by maboroshin** [comment]
@fast4x Will you give Proofreader privileges to all new translators in the future? It would be better to remove the approval by fast4x for all languages. There are also many people commenting on Crowdin about this approval issue in other languages.

Perhaps a simple word like "Play" is easier to lock. Otherwise, no approval would be better. Because, Crowdin has a voting system, but approval ignores this.

The Ukrainian "cancel" has collected one point and minus one vote, but you have approved the wrong string.
![image](https://github.com/user-attachments/assets/29b42718-ee9b-4e3a-b613-c489e0f0b77a)

----
## How to remove all approvals

1. "Side-by-side" can be selected in the upper right corner when viewing from a PC. A check box appears for those who have approval authority.

![image](https://github.com/user-attachments/assets/0f13a406-e9ed-4080-add4-e57d5d9761de)

![image](https://github.com/user-attachments/assets/e243e06d-3e6e-43a8-a669-3e59d84c69cc)
----

2. Using the Advanced Filter.

![image](https://github.com/user-attachments/assets/d2998edc-76be-4c55-b596-395ff3895667)

----

3. Display strings approved by fasxt4x.Press the "Filter" button.

![image](https://github.com/user-attachments/assets/0a654d70-f589-443b-a2a8-310b65167882)

----

4. Checks a target string. Or the top checkbox is all selections.

![image](https://github.com/user-attachments/assets/e243e06d-3e6e-43a8-a669-3e59d84c69cc)

----

5. Remove approval from 3 dot marks. Then press the "Save" button.

![image](https://github.com/user-attachments/assets/d649c3be-921f-4063-9889-339023d7bdaa)

**Comment by maboroshin** [comment]
Many issues remain.

Done:
![image](https://github.com/user-attachments/assets/0a245dfc-8023-472f-bb85-816869136787)

Hide:
![image](https://github.com/user-attachments/assets/48ffe680-afcd-4927-9110-e5a50d53be8e)
